server/apps/immich/src/controllers/person.controller.ts

```diff
@@ -39,4 +40,13 @@ export class PersonController {
   getPersonAssets(@Param() { id }: UUIDParamDto): Promise<AssetResponseDto[]> {
     return this.service.getAssets(id);
   }
+
+  @Put(':id')
+  update(
+    @GetAuthUser() authUser: AuthUserDto,
+    @Param() { id }: UUIDParamDto,
+    dto: UpdatePersonDto,
+  ): Promise<PersonResponseDto> {
+    return this.service.update(authUser.id, id, dto);
```

Comment:
We should follow the pattern we use everywhere else and pass the entire auth user object.